ticketutil/servicenow.py

```diff
@@ -365,6 +364,14 @@ def _prepare_ticket_fields(self, fields):
         return fields
 
 
+def devops_one_url(server, table, sys_id):
```

Comment:
It's a helper function which creates URL for DevOps One application.[1] I didn't feel it would be good idea to include it in the class itself, but it's related. I can add an example to /examples 
[1] https://redhat.service-now.com/pnt